mu-plugins/central-hub/tests/phpunit/unit/meta-data/isOkayToSaveMetaBox.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Tests for the function is_okay_to_save_meta_box().
+ *
+ * @package     spiralWebDb\centralHub\Tests\Unit\Metadata
+ * @since       1.3.0
+ * @author      Robert Gadon <rgadon107>
+ * @link        https://github.com/rgadon107/cornerstone
+ * @license     GNU-2.0+
+ */
+
+namespace spiralWebDb\centralHub\Tests\Unit\Metadata;
+
+use Brain\Monkey;
+use function spiralWebDB\Metadata\is_okay_to_save_meta_box;
+use spiralWebDb\Cornerstone\Tests\Unit\Test_Case;
+
+/**
+ * Class Tests_RegisterMetaBoxes
+ *
+ * @package spiralWebDb\centralHub\Tests\Unit\Metadata
+ * @group   meta-data
+ */
+class Tests_IsOkayToSaveMetaBox extends Test_Case {
+
+	/**
+	 * Prepare the test environment before each test.
+	 */
+	protected function setUp() {
+		parent::setUp();
+
+		require_once CENTRAL_HUB_ROOT_DIR . '/src/meta-data/meta-box.php';
+	}
+
```

Comment:
@rgadon107 Add a `tearDown` method to reset `$_POST` to an empty array. In doing so, `$_POST` is cleared out (1) after each test runs in this test case and (2) after this test case runs (ie for other tests).

Reply:
Done.